components/items/ItemsGrid/useNftActions.ts

```diff
@@ -172,6 +172,7 @@ export function useNftActions(entity: TokenEntity) {
 
   return {
     isOwner,
+    cheapestNFT,
```

Comment:
`getNftForBuying` got you covered.
no need for additional return

Reply:
hmm I dont see it 

solutions

1 - having  getNftForBuying and await inside the computed 
2 - creating a new variable and getting the cheapese nft using getNftForBuying, why not expose in the first place

Reply:
![image](https://github.com/kodadot/nft-gallery/assets/22791238/ce7a66cc-a104-468a-a2f8-d8c890b4ee1f)

Reply:
> 1 - having getNftForBuying and await inside the computed

awaiting inside the computed would not do anything, still will return a promise


> 2 - creating a new ref and getting the cheapese nft using getNftForBuying, why not expose in the first place , it's the same

not quite the same `getNFTFotBuying` will fetch only once

Reply:
> > 2 - creating a new ref and getting the cheapese nft using getNftForBuying, why not expose in the first place , it's the same
> 
> sure, but why create duplicate api surface?

ic got it